node/npm/glide/index.ts

```diff
@@ -185,19 +185,17 @@ function initialize() {
         SignedEncoding,
         UnsignedEncoding,
         UpdateByScore,
-        createLeakedArray,
-        createLeakedAttribute,
-        createLeakedBigint,
-        createLeakedDouble,
-        createLeakedMap,
-        createLeakedString,
-        parseInfoResponse,
         Script,
         ObjectType,
         ClusterScanCursor,
         BaseClientConfiguration,
         GlideClusterClientConfiguration,
         LevelOptions,
+        ReturnTypeRecord,
+        ReturnTypeMap,
+        ClusterResponse,
+        ReturnTypeAttribute,
+        PubSubSubscriptions,
```

Comment:
Remove.  PubSubSubscriptions is already exported as part of their respective namespaces.